### ISSUE TITLE
"make install" does *not* install the static libs

### ISSUE THREAD

**Issue by andrew-wja** [xianyi/OpenBLAS#2035]
Build commands:

```
pkgdir="/tmp/test-openblas/"
_config="BINARY=64 CC=aarch64-linux-gnu-gcc FC=aarch64-linux-gnu-gfortran HOSTCC=gcc TARGET=ARMV8 \
  USE_OPENMP=1 USE_THREAD=1 USE_COMPILER_TLS=0 \
  NO_STATIC=0 NO_SHARED=0 \
  NO_LAPACK=0 BUILD_LAPACK_DEPRECATED=1 \
  MAKE_NB_JOBS=${_ncpus} \
  PREFIX=/usr/aarch64-linux-gnu "

make ${_config} CFLAGS="-mtune=generic -I /usr/aarch64/include -L /usr/aarch64/lib -L usr/aarch64/lib64" libs netlib shared

make ${_config} DESTDIR="${pkgdir}" install

```

The resulting files in `/tmp/test-openblas` have the shared libs, but *not* the static libs, even though the static libs are built, and I can see the libopenblas-<target>.a file in the build directory.

**Comment by martin-frbg** [comment]
Which version is this, and are you using plain make or cmake ?

**Comment by andrew-wja** [comment]
Oh, sorry, I forgot the version number! It's the v0.3.4 tag. I am using plain make.

**Comment by martin-frbg** [comment]
Seems to work fine for me with current develop (at least judging from the output of a `make -n install`). Is that a typo in your first message, or do you actually get a "libopenblas-.a" rather than libopenblas-armv8p-r0.3.4.a ?

**Comment by martin-frbg** [comment]
On the other hand, specifiying any of the NO_something=0 may still be dangerous (despite some fixes made in response to #1422). It is generally better to leave them out as some places only (used to?) check if such a variable is defined at all, disregarding the value.
